.github/workflows/publish-major-minor.yml

```
@@ -0,0 +1,62 @@
+name: "Publish Major-Minor-Tags"
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  push-tags:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
```

I guess it's time to bump this
```suggestion
      - name: Set up Python 3.9
        uses: actions/setup-python@v2
        with:
          python-version: 3.9
```